scipy/stats/stats.py

```diff
@@ -1506,12 +1524,30 @@ def skewtest(a, axis=0, nan_policy='propagate'):
           * 'raise': throws an error
           * 'omit': performs the calculations ignoring nan values
 
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+          * 'two-sided':
+                p-value is calculated as :math:`2Pr(z \\ge |Z|)`
+                under the null hypothesis.
+          * 'less':
+                p-value is calculated as :math:`Pr(z \\le Z)` under the
+                null hypothesis.
+          * 'greater':
+                p-value is calculated as :math:`Pr(z \\ge Z)` under
+                the null hypothesis.
+
+        where :math:`Z` is the calculated value of the statistic.
```

Comment:
This is better. 
Just as we define Z as the observed value of the test statistic, z would need to be defined as the test statistic under the null hypothesis. 
Ideally, wouldn't we also tell the user what the test statistic for this test is? Would they know without looking at the source code?
On the other hand, I'm not sure we want to write it in such a way that the reader needs to be familiar with the z score test statistic. For instance, in the t-test, we would write about the difference in means (not the t statistic), and in kstest we write about the difference in CDF (not the precise statistic).
@chrisb83 how do you think we should document what alternatives mean, in general?

Reply:
> If you agree, I am not sure that we should leave this sort of thing for future work. The more I work with stats, the more I think that if we're going to make improvements like this, we should do the full job.

I agree with point 2). Yeah, we can leave this idea to a future PR that cleans up all the documentation in stats. For now, I will just define what z is. Is that ok?

Reply:
Thanks. Sorry it's not as simple as you originally planned! But I think it will be much more useful when you're done.

Reply:
> Thanks. Sorry it's not as simple as you originally planned! But I think it will be much more useful when you're done.

No worries, @mdhaber. I have attempted to change the documentation to be more intuitive. And yes, it does sound way more useful than it originally did. Even R prints the alternative hypothesis when a test is run. So, this change is definitely worth it.